app/build.gradle

```diff
@@ -9,7 +9,7 @@ android {
     defaultConfig {
         applicationId "com.example.nerdlauncher"
         minSdk 21
-        targetSdk 32
+        targetSdk 29
```

Comment:
Is there a reason why we are not using the latest targetSdk?

Reply:
I changed the targetSdk to 29 because on the targetSdk 30 or above it showed us only 3 activities available. Also a solution was to introduce <queries> in Android Manifest. What do you think is the better for us to use?